### ISSUE TITLE
lack of a changelog

### ISSUE THREAD

**Issue by iamstarkov** [mdx-js/mdx#528]
### my build is failing when I try to upgrade from 0.20.3 to v1 and I can't find change log to fix the problem

I have the blog built with next.js and mdx https://github.com/iamstarkov/iamstarkov.com/pull/69.

I have renovatebot and now integrations, so when renovate bot submitted mdx v1 bump PR, now stopped building it. I mean its fine, because its a major bump and I do expect some breaking changes affecting my blog.

But then I went to investigate what is in the v1 release and what are the breaking changes:
* releases page is empty https://github.com/mdx-js/mdx/releases
* changelog.md refers to GitHub releases https://github.com/mdx-js/mdx/blob/master/changelog.md

I sense catch 22 problem here.

### Your environment

N/A

### Steps to reproduce

https://github.com/iamstarkov/iamstarkov.com/

* mdx pre v1 works in master
* mdx@v1+ breaks in the PR

### Expected behaviour

Any changelog/release notes available

### Actual behaviour

No changelog/release notes available

Bottom line: I dont mind breaking changes in v1. it is expected. I want to know what are the changes and easy way to discover release notes

**Comment by macklinu** [comment]
Taking a look at the [v1.0.0 release on GitHub](https://github.com/mdx-js/mdx/releases/tag/v1.0.0), there is a link to the [migrating from v0 to v1 guide](https://mdxjs.com/migrating/v1/) on the MDX website. I used this to upgrade a Gatsby MDX v0 starter to use MDX v1 successfully the other day, so I hope this guide finds you well in your Next.js + MDX upgrade. 😄

**Comment by iamstarkov** [comment]
hmm 🤔 
I swear GitHub releases were empty when I filed an issue and GitHub doesnt allow to see an edit history.

im a bit confused. @johno, did you update GitHub release notes recently (after I filed an issue)?

**Comment by wooorm** [comment]
John has worked on this since the issue, yes! Not sure if he forgot to reply here or if he’s still working on it!

**Comment by iamstarkov** [comment]
there is no @mdx-js/mdx@1.0.0 published on npm, but there is one on GitHub 🤔

```
☯ npm info @mdx-js/mdx versions | grep "1.0.0"
  '1.0.0-alpha.0',
  '1.0.0-alpha.1',
  '1.0.0-alpha.5',
  '1.0.0-alpha.6',
  '1.0.0-alpha.7',
  '1.0.0-alpha.8',
  '1.0.0-alpha.9',
  '1.0.0-alpha.10',
  '1.0.0-alpha.11',
  '1.0.0-alpha.12',
  '1.0.0-alpha.13',
  '1.0.0-alpha.16',
  '1.0.0-alpha.17',
  '1.0.0-ci.0',
  '1.0.0-rc.0',
  '1.0.0-rc.2',
  '1.0.0-rc.3',
  '1.0.0-rc.4',
```

**Comment by iamstarkov** [comment]
@wooorm, I see. 

@johno, thank you

**Comment by wooorm** [comment]
Hmm, the version does show on https://www.npmjs.com/package/@mdx-js/mdx!

**Comment by johno** [comment]
@iamstarkov I filled in the patch releases because they were indeed missing. I also added a direct link to the migration guide for v1.0.0 since you had to link into the release post to find the migration guide. After doing so I completely spaced commenting back on this and closing the issue! 😬 

Apologies for any confusion and thanks for raising this issue @iamstarkov!

**Comment by iamstarkov** [comment]
@woorm unless, my sight degraded even more I cant see 1.0.0 https://www.npmjs.com/package/@mdx-js/mdx?activeTab=versions

**Comment by iamstarkov** [comment]
@johno its cool. thanks for referencing the migration guide

**Comment by wooorm** [comment]
@iamstarkov Nope, it’s my glasses that aren’t good enough apparently!

<img width="764" alt="Screenshot 2019-04-18 at 3 24 15 pm" src="https://user-images.githubusercontent.com/944406/56364084-08d81380-61ee-11e9-8a52-c35b99edac1b.png">

You’re right, `1.0.0` is missing from npm. @johno I guess we should still publish it?

**Comment by iamstarkov** [comment]
thank you for [`examples/next`](https://github.com/mdx-js/mdx/blob/master/examples/next/).
it is more helpful than [next's with-mdx example](https://github.com/zeit/next.js/tree/canary/examples/with-mdx) because of de-synced versions. 

if I may, I would suggest to deprecate next's example and point to mdx's one
